[dim order]  solve broken doc

Gasoonjia opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #146641

Differential Revision: [D69265340](https://our.internmc.facebook.com/intern/diff/D69265340/)

Gasoonjia committed:
[dim order]  solve broken doc



Differential Revision: [D69265340](https://our.internmc.facebook.com/intern/diff/D69265340/)

[ghstack-poisoned]

Gasoonjia commented:
@pytorchbot label "topic: not user facing"

svekars commented on torch/_tensor.py:
```diff
@@ -1517,6 +1517,7 @@ def dim_order(
         Args:
             ambiguity_check (bool or List[torch.memory_format]): The check method for ambiguity of dim order.
 
+        Examples::
```
```suggestion
        Examples::
        
```

Gasoonjia committed:
Update on "[dim order]  solve broken doc"


Differential Revision: [D69265340](https://our.internmc.facebook.com/intern/diff/D69265340/)

[ghstack-poisoned]

Gasoonjia committed:
Update on "[dim order]  solve broken doc"


Differential Revision: [D69265340](https://our.internmc.facebook.com/intern/diff/D69265340/)

[ghstack-poisoned]
